### PR TITLE
two of four docs slashed and burned

### PR DESCRIPTION
**Pull request by sarah11918**

### CODE REVIEW

File: docs/XElement/sarahEdits/Overview.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+XElement allows you to create dynamic HTML elements with interactivity (e.g. animations, transitions, event listeners) natively in an Astro page or component... without the need for writing and importing a component in one of the many JavaScript frameworks Astro supports (React, Vue, Svelte, Solid, Preact).
+
+XElement web components give you the option to choose *when* and *how* to run JavaScript or TypeScript on the client-side on a per-component basis, similar to the way Astro controls the hydration of framework components via `client:*` directives.
+
+Unlike other framework components, XElement-generated components can even contain other framework components. This allows you to combine the outputs of otherwise isolated islands of interactivity.
+
+----------
+
+### To use XElement components in your project, follow these steps:
```
Comment (aFuzzyBear) [inline docs/XElement/sarahEdits/Overview.md:13]:
```suggestion
## Getting Started:
```

File: docs/XElement/sarahEdits/GettingStarted.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+## `<Counter>`
+
+How could we not provide an example of a 'Counter'?
+
+You will notice that Astro starter "framework" examples (e.g. React, Svelte etc.) contain a counter component to show you how framework components work in Astro.
+
+With XElement, you can have a fully-functioning, interactive *Astro* counter component, no other framework required!
+
+
+## `<Counter>`
```
Comment (aFuzzyBear) [inline docs/XElement/sarahEdits/GettingStarted.md:77]:
```suggestion
### Let Start
```

File: docs/XElement/sarahEdits/GettingStarted.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+Let's create our own interactive `<Counter>` component. Note that this time, we will need to import multiple XElement components, one to represent each HTML element we wish to create.
```
Comment (aFuzzyBear) [inline docs/XElement/sarahEdits/GettingStarted.md:79]:
```suggestion
Let's create our own interactive `<Counter>` component. Note that this time, we will need to declare a few different XElements, one to represent each HTML element we wish to create.
```

File: docs/XElement/sarahEdits/GettingStarted.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    <Button 
+        @click={(event,store)=>{
+        display.textContent = --store.count
+    }}>
+        -
+    </Button>
+</Counter>
+```
+### `@` decorators
+To apply JavaScript to our component, we use one of the many `@` decorators recognized by `XElement` to specify what type of action we wish it to perform.
```
Comment (aFuzzyBear) [inline docs/XElement/sarahEdits/GettingStarted.md:127]:
```suggestion
To start enhancing our component, we use one of the many `@` decorators recognized by `XElement` to specify what type of action we wish it to perform.
```

File: docs/XElement/sarahEdits/GettingStarted.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+### `store`
+
+XElement also provides a `store`: a special non-persistent data object that is available to all `XElement` components. This lets you *store* your data and allows it to be used elsewhere on the page. We are keeping track of our counter's `count` in this store object.
+
+### `id`
+
+Giving an `id` to an XElment, as we have done with `<Display>`, allows us to target that element from elsewhere in the component tree. In this example, we can update the counter's displayed count by referencing, then changing the `<Display>` element's `textContent`.
+
+### Summary 
+In this example, what we are asking our `<Counter>` parent element to **do**  is initialise the `store` with a `count` of `0`.
```
Comment (aFuzzyBear) [inline docs/XElement/sarahEdits/GettingStarted.md:140]:
```suggestion
In this example, what we are asking our  parent element, the `<Counter>` to **do**  is initialise the `store` with a `count` of `0`. 
```

File: docs/XElement/sarahEdits/GettingStarted.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+### Summary 
+In this example, what we are asking our `<Counter>` parent element to **do**  is initialise the `store` with a `count` of `0`. 
+
+And, we are telling the buttons that when they receive a `click` event, they should increment or decrement the `store.count` and update the `display.textContent`.
+
+This `<Counter>` example renders the following HTML to the page:
+```html
+<counter>
+    <span id="display">0</span>
```
Comment (aFuzzyBear) [inline docs/XElement/sarahEdits/GettingStarted.md:147]: This is missing the second button 🙂

Comment (sarah11918) [reply docs/XElement/sarahEdits/GettingStarted.md:147]: D'oh! I was experimenting moving them around to maybe put the buttons on either side of the display, and one of them obviously never got pasted back in.

**Review by aFuzzyBear — APPROVED** [review]
This looks amazing 💚
Thank you buddy !!

**Pull request merged**